diff --git a/CMakeLists.txt b/CMakeLists.txt
index 554362bb..0cac9c42 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2063,6 +2063,18 @@ INSTALL(
   COMPONENT headers
   )
 
+add_library(netCDF::netcdf ALIAS netcdf)
+target_include_directories(netcdf
+    PUBLIC
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
+)
+INSTALL(EXPORT netCDFTargets
+    FILE netCDFTargets.cmake
+    NAMESPACE netCDF::
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/netCDF
+)
+
 # Create export configuration
 write_basic_package_version_file(
   "${CMAKE_CURRENT_BINARY_DIR}/netCDF/netCDFConfigVersion.cmake"
diff --git a/netCDFConfig.cmake.in b/netCDFConfig.cmake.in
index c89cf3e6..3f9046ed 100644
--- a/netCDFConfig.cmake.in
+++ b/netCDFConfig.cmake.in
@@ -40,3 +40,7 @@ set(netCDF_HAS_DAP4 @HAS_DAP4@)
 set(netCDF_HAS_DISKLESS @HAS_DISKLESS@)
 set(netCDF_HAS_MMAP @HAS_MMAP@)
 set(netCDF_HAS_JNA @HAS_JNA@)
+
+if (NOT TARGET netCDF::netcdf)
+    include("${netCDF_CMAKE_DIR}/netCDFTargets.cmake")
+endif()
